<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>DeleteDialogBase</class>
  <widget class="QWidget" name="DeleteDialogBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>542</width>
        <height>374</height>
      </rect>
    </property>
    <property name="minimumSize" >
      <size>
        <width>420</width>
        <height>320</height>
      </size>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="ddWarningIcon" >
              <property name="sizePolicy" >
                <sizepolicy>
                  <hsizetype>4</hsizetype>
                  <vsizetype>4</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text" >
                <string>Icon Placeholder, not in GUI</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="ddDeleteText" >
                  <property name="text" >
                    <string>Deletion method placeholder, never shown to user.</string>
                  </property>
                  <property name="alignment" >
                    <set>Qt::AlignCenter</set>
                  </property>
                  <property name="wordWrap" >
                    <bool>true</bool>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="K3ListBox" name="ddFileList" >
          <property name="selectionMode" >
            <enum>NoSelection</enum>
          </property>
          <property name="toolTip" stdset="0" >
            <string>List of files that are about to be deleted.</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>This is the list of items that are about to be deleted.</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="ddNumFiles" >
          <property name="text" >
            <string>Placeholder for number of files, not in GUI</string>
          </property>
          <property name="alignment" >
            <set>Qt::AlignVCenter|Qt::AlignRight</set>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="ddShouldDelete" >
          <property name="text" >
            <string>&amp;Delete files instead of moving them to the trash</string>
          </property>
          <property name="toolTip" stdset="0" >
            <string>If checked, files will be permanently removed instead of being placed in the Trash Bin</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>&lt;qt>&lt;p>If this box is checked, files will be &lt;b>permanently removed&lt;/b> instead of being placed in the Trash Bin.&lt;/p>

&lt;p>&lt;em>Use this option with caution&lt;/em>: Most filesystems are unable to reliably undelete deleted files.&lt;/p>&lt;/qt></string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <customwidgets>
    <customwidget>
      <class>K3ListBox</class>
      <extends>Q3ListBox</extends>
      <header>k3listbox.h</header>
    </customwidget>
  </customwidgets>
  <connections>
    <connection>
      <sender>ddShouldDelete</sender>
      <signal>toggled(bool)</signal>
      <receiver>DeleteDialogBase</receiver>
      <slot>slotShouldDelete(bool)</slot>
    </connection>
  </connections>
</ui>
